packages/vm/src/evm/opcodes/functions.ts

```diff
@@ -70,7 +71,7 @@ export const handlers: Map<number, OpHandler> = new Map([
     0x03,
     function (runState) {
       const [a, b] = runState.stack.popN(2)
-      const r = a.sub(b).toTwos(256)
+      const r = (a - b) & TWO_POW256_BIGINT
```

Comment:
This uses and while others use mod.

Reply:
Yep, I was following PyEVM here - https://github.com/ethereum/py-evm/blob/1af151ab218b905f4fdf7a285cbe14ebf094a7c4/eth/vm/logic/arithmetic.py#L85 since we don't have a native twosComplement function with bigints.  Just experimenting for now but will see what results it provides.